nvm.sh

```diff
@@ -1870,6 +1870,7 @@ nvm() {
           echo "$NVM_DIR/*/lib/node_modules removed from \$NODE_PATH"
         fi
       fi
+      unset NVM_BIN NVM_PATH
```

Comment:
`-f` is useful to remove function definition.
And I do not see any problem if it unset a variable that no dot exists.
```bash
~> export OI=123
~> echo $?
0
~> unset OI
~> echo $?
0
~> unset OI
~> echo $?
0
```